### PR TITLE
Dashboard page, simulator instructions

### PR DESCRIPTION
**Pull request by jsani-radar**
## What?

https://app.clubhouse.io/radarlabs/story/6708/mock-tracking-external-documentation

## Why?

Simulator launch 7/22

## Screenshots (optional)

Screen Shot 2021-07-22 at 12 32 20 AM

## Anything Else? (optional)

**Not in scope** - writing up all dashboard functionality

### CODE REVIEW

File: docs/dashboard.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+sidebar_position: 12
+title: Dashboard
+id: dashboard
+---
+
+Use the Radar [dashboard](https://radar.io/dashboard) to manage, explore, and analyze your Radar data, such as geofences, users, events, and trips.
+
+The [dashboard](https://radar.io/dashboard) can also be used to configure [integrations](https://radar.io/dashboard/integrations), [project settings](https://radar.io/dashboard/settings) such as enabled features (ex. places) and event types (ex. `user.exited_place`), and [custom data retention windows](https://radar.io/dashboard/privacy).
```
Comment (corypisano) [inline docs/dashboard.mdx:9]: nit: i think typically I've seen `e.g.` in our docs vs ex https://radar.io/documentation/geofences#create-geofences

**Review by andlum — APPROVED** [review]
Looks good @jsani-radar!

File: docs/dashboard.mdx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+sidebar_position: 12
+title: Dashboard
+id: dashboard
+---
+
+Use the Radar [dashboard](https://radar.io/dashboard) to manage, explore, and analyze your Radar data, such as [geofences](/geofences), [users](https://radar.io/dashboard/users), [events](https://radar.io/dashboard/events), and [trips](/trip-tracking).
+
+The [dashboard](https://radar.io/dashboard) can also be used to configure [integrations](https://radar.io/dashboard/integrations), [project settings](https://radar.io/dashboard/settings) such as enabled features (e.g. [places](/places)) and event types (ex. `user.exited_place`), and [custom data retention windows](https://radar.io/dashboard/privacy).
```
Comment (nickpatrick) [inline docs/dashboard.mdx:9]: Nit: `e.g.,` (with a comma) in both places instead of `ex.`

**Comment by nickpatrick** [comment]
Excited to finally have a dashboard docs page again!

**Review by tjulien — APPROVED** [review]
Looks great!

**Pull request merged**